p-token/src/entrypoint-runtime-verification.rs

```diff
@@ -360,6 +360,79 @@ fn get_rent(account_info: &AccountInfo) -> &Rent {
     }
 }
 
+/// This function encapsulates the specification of validating the signature requirements
+/// In particular, code from mod.rs::validate_owner is checked
+#[inline(always)]
```

Comment:
```suggestion
#[inline(never)]
```

I am adding cut point rules for function calls so I would like this to cut here and that means it can't be inlined